dex/pair/src/lib.rs

```diff
@@ -148,6 +151,8 @@ pub trait Pair<ContractReader>:
         let lpt = context.get_lp_token_id();
         let liq_added = context.get_liquidity_added();
         self.send().esdt_local_mint(lpt, 0, liq_added);
+
+        self.update_safe_state_from_context(&mut context);
```

Comment:
clippy thinks you don't need `&mut` here, only `&context`

Reply:
done